Prog/reshape ACS data for population projection.do

```diff
@@ -0,0 +1,568 @@
+rename HC02_EST_VC01 totmalepop
+rename HC03_EST_VC01 totfemalepop
+
+xpose, clear varname format
+
+order _varname v1-v11
+
+save "D:\DCData\Libraries\RegHsg\Prog\ACS_1year_gender.dta"
+
+export excel using "D:\DCData\Libraries\RegHsg\Prog\ACS_1year_gender.xlsx"
+
+clear
+
+***************************population by gender 5 year estimate ****************
+
+import excel "L:\Libraries\RegHsg\Raw\ACS 5 year estimate\All_5year.xlsx", sheet("Sheet1") firstrow
+
+label variable HC01_EST_VC01 "totpop"
+label variable HC02_EST_VC01 "mtotpop"
+label variable HC03_EST_VC01 "ftotpop"
+label variable HC01_EST_VC03 "totunder5"
+label variable HC02_EST_VC03 "mtotunder5"
+label variable HC03_EST_VC03 "ftotunder5"
+label variable HC01_EST_VC04 "tot5to9"
+label variable HC02_EST_VC04 "mtot5to9"
+label variable HC03_EST_VC04 "ftot5to9"
+label variable HC01_EST_VC05 "tot10to14"
+label variable HC02_EST_VC05 "mtot10to14"
+label variable HC03_EST_VC05 "fot10to14"
+label variable HC01_EST_VC06 "tot15to19"
+label variable HC02_EST_VC06 "mtot15to19"
+label variable HC03_EST_VC06 "ftot15to19"
+label variable HC01_EST_VC07 "tot20to24"
+label variable HC02_EST_VC07 "mtot20to24"
+label variable HC03_EST_VC07 "ftot20to24"
+label variable HC01_EST_VC08 "tot25to29"
+label variable HC02_EST_VC08 "mtot25to29"
+label variable HC03_EST_VC08 "ftot25to29"
+label variable HC01_EST_VC09 "tot30to34"
+label variable HC02_EST_VC09 "mtot30to34"
+label variable HC03_EST_VC09 "ftot30to34"
+label variable HC01_EST_VC10 "tot35to39"
+label variable HC02_EST_VC10 "mtot35to39"
+label variable HC03_EST_VC10 "ftot35to39"
+label variable HC01_EST_VC11 "tot40to44"
+label variable HC02_EST_VC11 "mtot40to44"
+label variable HC03_EST_VC11 "ftot40to44"
+label variable HC01_EST_VC12 "tot45to49"
+label variable HC02_EST_VC12 "mtot45to49"
+label variable HC03_EST_VC12 "ftot45to49"
+label variable HC01_EST_VC13 "tot50to54"
+label variable HC02_EST_VC13 "mtot50to54"
+label variable HC03_EST_VC13 "ftot50to54"
+label variable HC01_EST_VC14 "tot50to59"
+label variable HC02_EST_VC14 "mtot50to59"
+label variable HC03_EST_VC14 "ftot50to59"
+label variable HC01_EST_VC15 "tot60to64"
+label variable HC02_EST_VC15 "mtot60to64"
+label variable HC03_EST_VC15 "ftot60to64"
+label variable HC01_EST_VC16 "tot65to69"
+label variable HC02_EST_VC16 "mtot65to69"
+label variable HC03_EST_VC16 "ftot65to69"
+label variable HC01_EST_VC17 "tot70to74"
+label variable HC02_EST_VC17 "mtot70to74"
+label variable HC03_EST_VC17 "ftot70to74"
+label variable HC01_EST_VC18 "tot75to79"
+label variable HC02_EST_VC18 "mtot75to79"
+label variable HC03_EST_VC18 "ftot75to79"
+label variable HC01_EST_VC19 "tot80to84"
+label variable HC02_EST_VC19 "mtot80to84"
+label variable HC03_EST_VC19 "ftot80to84"
+label variable HC01_EST_VC20 "tot85up"
+label variable HC02_EST_VC20 "mtot85up"
+label variable HC03_EST_VC20 "ftot85up"
+
+
+gen totunder5= HC01_EST_VC03*HC01_EST_VC01/100
+gen mtotunder5= HC02_EST_VC03*HC01_EST_VC01/100
```

Comment:
i don't think these are all right. HC02_EST_VC03 is percent of kids under 5 not number of males under 5 according to the metadata.